avocado/plugins/run.py

```diff
@@ -253,7 +253,7 @@ def configure(self, parser):
                                  help_msg=help_msg,
                                  choices=('none', 'stdout', 'stderr',
                                           'both', 'combined', 'all'),
-                                 parser=parser,
+                                 parser=out_check,
                                  default=None,
```

Comment:
How is this change related to the current group of fixes? Do you mind elaborating with an extra sentence or two?

Reply:
Oh, this is just a usability fix related to Avocado jobs, that is, to `avocado run --help`.  It's not related to the nrunner though.

Reply:
I see. My typical expectation when looking at a PR would be to have all changes in the file diff correspond to the topic of the PR though. I guess this is just a hit and miss situation.